src/components/va-pagination/va-pagination.e2e.ts

```diff
@@ -0,0 +1,44 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+import { axeCheck } from '../../testing/test-helpers';
+
+describe('va-pagination', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<va-pagination></va-pagination>');
+    const element = await page.find('va-pagination');
+    expect(element).toEqualHtml(`
+      <va-pagination page="1" class="sc-va-pagination-h sc-va-pagination-s hydrated">
```

Comment:
I know `hydrated` is a stencil thing, but I'm not sure where these other classes are coming from or why they are there.